image-pipeline/expression-scatter-plots.js

```diff
@@ -20,87 +21,280 @@ const options = {
 }
 const { values } = parseArgs({ args, options })
 
-// Make `images` directory if absent
-access('images', async err => {
-  if (err) {
-    await mkdir('images')
-  }
-})
+// Candidates for CLI argument
+// CPU count on Intel i7 is 1/2 of reported, due to hyperthreading
+const numCPUs = os.cpus().length / 2 - 1
```

Comment:
Non-blocking: shouldn't this also take into account newer m1/m2 chips?  Or is the calculation the same?

Reply:
Good point!  Running on Emily's M1 Pro reports: `Number of CPUs to be used on this client: 4`, so I think this is OK for now.  Ultimately, I imagine we'll drop the `/ 2` for the default value and add a CLI argument `--num-cpu` to override that.